lib/membrane/core/child/pads_specs.ex

```diff
@@ -260,25 +256,8 @@ defmodule Membrane.Core.Child.PadsSpecs do
     end
   end
 
-  defp generate_pad_property_doc(:caps, caps) do
-    caps
-    |> Bunch.listify()
-    |> Enum.map(fn
-      {module, params} ->
-        params_doc =
-          Enum.map_join(params, ",<br/>", fn {k, v} ->
-            Bunch.Markdown.hard_indent("<code>#{k}: #{inspect(v)}</code>")
-          end)
-
-        "<code>#{inspect(module)}</code>, restrictions:<br/>#{params_doc}"
-
-      module ->
-        "<code>#{inspect(module)}</code>"
-    end)
-    ~> (
-      [doc] -> doc
-      docs -> docs |> Enum.join(",<br/>")
-    )
+  defp generate_pad_property_doc(:caps_pattern, pattern_string) when is_binary(pattern_string) do
+    "<code>#{pattern_string}</code>"
```

Comment:
````suggestion
    """
    ```
    #{pattern_string}
    ```
    """
````

Also, it would be better if a list of caps generated a code block for each pattern in the list

Reply:
Having ``` instead of `<code>` leads to such a warnings
```
warning: ExDoc.Markdown.Earmark (warning) test/support/caps_test/outer_sink_bin.ex:24 Failed to find closing <tr>
```